<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sala_de_espera</class>
 <widget class="QWidget" name="sala_de_espera">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sala de Espera</string>
  </property>
  <widget class="QLabel" name="jefe_partida">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>USUARIO   1   (JEFE)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   2</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   3</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuario_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USUARIO   4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_jugar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>510</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>JUGAR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="limpiessa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FERNANDO   LIMPIESSA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sin_rastro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>JAIME   SINRASTRO</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cervessa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FERNANDO   CERVESSA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="del_trio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FELIPE   DEL   TRIO</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="usain_nebolt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>USAIN   NEBOLT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nebcoins">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NEBCOINS</string>
   </property>
  </widget>
  <widget class="QLabel" name="poderes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PODERES</string>
   </property>
  </widget>
  <widget class="QLabel" name="usuarios">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>USUARIOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="velocidad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VELOCIDAD</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vel_rapida">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RAPIDA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vel_media">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MEDIA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vel_max">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MAXIMA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vel_baja">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BAJA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="naranjo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
border-color: rgb(52, 52, 52);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="verde">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rosado">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="morado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rojo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="celeste">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="color">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COLOR</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>16</width>
     <height>551</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>701</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>CHAT</string>
   </property>
   <widget class="QLineEdit" name="mensaje_chat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enviar_chat">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ENVIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="chat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>631</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ArcadeClassic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="cta_regresiva">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>490</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CUENTA   REGRESIVA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VOLVER</string>
   </property>
  </widget>
  <widget class="QLabel" name="color_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PUNTOS MAXIMOS</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="puntos_maximos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="counter">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>510</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
